fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindSink.java

```diff
@@ -167,7 +168,9 @@ public List<Rule> buildRules() {
                                 }
                             }
 
-                            if (!haveInputSeqCol && !isPartialUpdate) {
+                            if (!haveInputSeqCol && !isPartialUpdate && (
+                                    boundSink.getDmlCommandType() != DMLCommandType.UPDATE
+                                            && boundSink.getDmlCommandType() != DMLCommandType.DELETE)) {
```

Comment:
could we add a help function in DMLCommandType for easy use?

Reply:
sounds not necessary? only 4 types